app/services/witter.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const config = require('../../config');
+const logger = require('../logger');
+const errors = require('../errors');
+
+export async function weet() {
+  try {
+    const { data } = await axios.get(`${config.common.apiClient}/quotes/random`);
```

Comment:
This is not bad but it is much better if at the time of declaring the variable you do `const {apiClient} = require ('../../ config').common;`